addons/im_livechat/tests/test_discuss_channel.py

```diff
@@ -21,3 +22,26 @@ def test_unfollow_from_non_member_does_not_close_livechat(self):
         self.assertTrue(chat.livechat_active)
         chat.with_user(chat.livechat_operator_id.user_ids[0]).action_unfollow()
         self.assertFalse(chat.livechat_active)
+
+    def test_livechat_conversation_history(self):
+        """Test livechat conversation history formatting"""
```

Comment:
I will adapt this one in https://github.com/odoo/odoo/pull/210125 to include attachments.